pkgs/by-name/li/librsvg/package.nix

```diff
@@ -64,6 +64,16 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-vBu81BkSCwmNsovqVTNdneJHDU5qn27pcge0EPwVhn0=";
   };
 
+  patches = [
+    # too_many_elements test fails with libxml 2.15.0,
+    # because libxml2 no longer updates the element count
+    # before erroring out, which breaks the librsvg limit check.
+    #
+    # This is okay, because the error is still detected.
+    # The error is simply not reported accurately.
+    ./expect-any-error-too-many-elements.patch
```

Comment:
Please mention the upstream issue for future reference (I know you posted it, but still):

https://gitlab.gnome.org/GNOME/librsvg/-/issues/1201